pages/Christmas-Items.md

```diff
@@ -0,0 +1,48 @@
+Christmas items are [Christmas specialities](https://github.com/Slimefun/Slimefun4/wiki/Christmas-Seasonal-Category).<br>
+These items consist of mostly foods.
```

Comment:
Right now it kind of sounds like the items in the category are made of food.
Id propose something more along the lines of: 
`This category mostly consists of different kinds of food. `